### PR TITLE
Add DLA

### PR DESCRIPTION
**Pull request by yinglielalala**
DLA network: By aggregating information of different layers and blocks in a network, the performance of multiple tasks can be significantly improved, including image classification, segmentation, fine-grained classification, etc

### CODE REVIEW

File: DLA/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+```bash
+wget https://oneflow-public.oss-cn-beijing.aliyuncs.com/model_zoo/cv/classification/DLA/pretrain_model.zip
+unzip pretrain_model.zip
+```
+
+### Run Oneflow Training script
+We have installed visdom to visualize the training model, and run the following program to enter http://localhost:8097/ get the training curve.
+
+```
```
Comment (Ldpe2G) [inline DLA/README.md:25]:
```python

File: DLA/infer.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  unzip data.zip
+fi
+
+if [ ! -d "$PRETRAIN_MODEL_PATH" ]; then
+  wget https://oneflow-public.oss-cn-beijing.aliyuncs.com/model_zoo/cv/classification/DLA/pretrain_model.zip
+  unzip pretrain_model.zip
+fi
+
+python3 infer.py --model_path $PRETRAIN_MODEL_PATH --image_path $IMAGE_PATH
+
```
Comment (Ldpe2G) [inline DLA/infer.sh:19]: 去除多余空格

File: DLA/train_oneflow.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+    writer = open("of_losses.txt", "w")
+    for o in of_losses:
+        writer.write("%f\n" % o)
+    writer.close()
+
+if __name__ == "__main__":
+    args = _parse_args()
+    main(args)
+
```
Comment (Ldpe2G) [inline DLA/train_oneflow.py:136]: 去除多余空格

File: DLA/train_oneflow.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+python train_oneflow.py \
+    --save_checkpoint_path $CHECKPOINT_PATH \
+    --ofrecord_path $OFRECORD_PATH \
+    --learning_rate $LEARNING_RATE \
+    --mom $MOM \
+    --epochs $EPOCH \
+    --train_batch_size $TRAIN_BATCH_SIZE \
+    --val_batch_size $VAL_BATCH_SIZE \
+    # --load_checkpoint $LOAD_CHECKPOINT
+
```
Comment (Ldpe2G) [inline DLA/train_oneflow.sh:32]: 去除多余空行

File: DLA/README.md (hunk 1 above)
Comment (yinglielalala) [reply DLA/README.md:27]: 已添加

File: DLA/infer.sh (hunk 2 above)
Comment (yinglielalala) [reply DLA/infer.sh]: 已删除多余空行

File: DLA/train_oneflow.py (hunk 3 above)
Comment (yinglielalala) [reply DLA/train_oneflow.py]: 已删除多余空行

File: DLA/train_oneflow.sh (hunk 4 above)
Comment (yinglielalala) [reply DLA/train_oneflow.sh]: 已删除多余空行

**Review by Ldpe2G — APPROVED** [review]

**Pull request merged**